src/blueDarkTheme.ts

```diff
@@ -37,6 +37,8 @@ export const MD3BluiDarkTheme = {
         onBackground: BLUIColors.neutral[95],
         surface: BLUIColors.neutral[6],
         onSurface: BLUIColors.neutral[95],
+        surfaceDisabled: Color(BLUIColors.neutral[90]).alpha(0.2).rgb().string(),
+        // TODO : onSurfaceDisabled: 'Value to be updated',
```

Comment:
Hi @huayunh, can you suggest what should be the color value for `onSurfaceDisabled`? Also, what is the difference between surfaceDisabled and our `disabled` custom color? Or it is the same? 
Our figma color for custom `disabled` color.
<img width="444" alt="Screenshot 2023-12-13 at 3 27 19 PM" src="https://github.com/etn-ccis/blui-react-native-themes/assets/133877691/7af4a298-8ae3-42e0-94a0-1cb17eda2ea0">

Reply:
did you and Jeff work on the same thing? Is your PR related to https://github.com/etn-ccis/blui-react-native-themes/pull/68?

Reply:
> did you and Jeff work on the same thing? Is your PR related to #68?

I was fixing https://github.com/etn-ccis/blui-react-native-themes/issues/66. Checkbox disabled color using surfaceDisabled. https://github.com/callstack/react-native-paper/blob/c190a95e6b2b3842d372ba8f75477752d1042034/src/components/TextInput/Adornment/utils.ts#L13

Reply:
Yes, will close this PR.